src/client.ts

```diff
@@ -181,7 +181,11 @@ export class Client {
    * directory.
    */
   public start(): void {
-    if (this.config.isValid) {
+    if (process.env._APPSIGNAL_DIAGNOSE === "true") {
```

Comment:
We can also move this check up to the constructor line 148. We don't need to start a lot of the other things there, like: the metrics, OpenTelemetry and the minutely probes.